popUp-modal.css

```diff
@@ -0,0 +1,211 @@
+  cursor: pointer;
+  position: absolute;
+  top: 45px;
+  right: 60px;
+}
+
+.desktop-img {
+  width: 100%;
+}
+
+.modal-body {
+  padding: 0;
+}
+
+.title-container #title {
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 600;
+  color: #172b4d;
+  margin: 0;
+}
+
+#tech-desktop {
+  display: none;
+}
+
+#tech-mobile {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 10px;
+  padding: 0;
+  list-style: none;
+}
+
+#tech-mobile li {
+  font-size: 12px;
+  line-height: 16px;
+  font-weight: 600;
+  color: #3a4a42;
+  background: #ebf0ee;
+  border-radius: 4px;
+  padding: 8px 12px;
+}
+
+#projectDescription {
+  font-size: 16px;
+  line-height: 24px;
+  font-weight: 400;
+  color: #344563;
+  text-align: justify;
+}
+
+.modal-footer {
+  padding: 5px;
+  color: white;
+  display: flex;
+  gap: 10px;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+}
+
+.modal-footer .show-project-btn {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 5px;
+  background: #36b37f;
+  border-radius: 5px;
+  padding: 10px;
+  border: none;
+  width: 153px;
+  height: 48px;
+  color: #fff;
+  font-size: 17px;
+  line-height: 24px;
+  font-weight: 500;
+  letter-spacing: 0.03em;
+  cursor: pointer;
+}
+
+@media screen and (min-width: 768px) {
+  .content-modal {
+    margin: 20px auto;
+    width: 70%;
+    padding: 10px;
+  }
+
+  .close-btn {
+    float: right;
+    color: #67798e;
+    cursor: pointer;
+    position: static;
+    font-weight: 400;
+    top: 0;
+    right: 0;
+    background: #ebecf0;
+    padding: 0 8px;
+    border-radius: 10px;
+    margin-bottom: 8px;
+  }
+
+  .desktop-img {
+    width: 100%;
+    height: 550px;
+  }
+
+  .modal-body {
+    padding: 10px;
+  }
+
+  .title-container #title {
+    font-size: 32px;
+    line-height: 44px;
+    font-weight: 700;
+    color: #172b4d;
+    width: 600px;
+    margin: 0;
+  }
+
+  #tech-desktop {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    gap: 10px;
+    padding: 0;
+    list-style: none;
+  }
+
+  #tech-desktop li {
+    font-size: 12px;
+    line-height: 16px;
+    font-weight: 600;
+    color: #344563;
+    background: #ebecf0;
+    border-radius: 8px;
+    padding: 8px 12px;
+  }
+
+  #tech-mobile {
+    display: none;
+  }
+
+  #projectDescription {
+    font-size: 16px;
+    line-height: 30px;
+    font-weight: 400;
+    color: #344563;
+    text-align: justify;
+  }
+
+  .modal-footer {
+    position: absolute;
+    bottom: -50px;
+    right: 15%;
+    gap: 12px;
+  }
+
+  .modal-footer .show-project-btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 5px;
+    background: #36b37f;
+    border-radius: 5px;
+    padding: 12px;
+    border: none;
+    width: 141px;
+    height: 48px;
+    color: #fff;
+    font-size: 15px;
+    line-height: 20px;
+    font-weight: 500;
+    letter-spacing: 0.03em;
+  }
+}
```

Comment:
[OPTIONAL] Kindly ensure that the buttons of the modal (`see live` and `see source`) are well positioned and matches that of the figma design since there are some disparities currently. :wink: 

![Screenshot from 2023-02-02 12-41-47](https://user-images.githubusercontent.com/65233860/216328495-b21a8886-de24-432e-90fb-287ca172089c.png)